### ISSUE TITLE
Non-Cors Custom server?

### ISSUE THREAD

**Issue by consciousweb** [CulturalMe/meteor-slingshot#240]
Im trying to connect to a s3 nodechef object server that does not support cors configurations, and does not use regions, just bucket + endpoint + filename and im receiving No 'Access-Control-Allow-Origin' header error. Is there anyway to disable cors requirements of slingshot to get past this?

**Comment by adammoisa** [comment]
+1 @consciousweb. Did you ever figure this out?

**Comment by consciousweb** [comment]
Nope, this package has been abandoned for 5 years now so with no response I decided it was not worth my time and found other packages to accomplish what i needed

**Comment by evolross** [comment]
What are the other packages you've found that maintained?

Our production app still uses this package and it works. And we use `browser-policy`.

**Comment by adammoisa** [comment]
@consciousweb +1 what are the other packages that worked for you?

**Comment by consciousweb** [comment]
fs, aws-sdk/clients/s3, meteor/ostrio:files

http upload -> temp mongo storage -> s3 upload
